src/components/week/LoggedInUserWeekList.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import styled from "styled-components"
+import { listLikedContents } from "../../graphql/queries"
+import { API, graphqlOperation } from "aws-amplify"
+import { useQuery } from "react-query"
+import { ContentfulWeek } from "../../../graphql-types"
+import WeekCard from "./WeekCard"
+
+interface Props {
+  data: any
+}
+
+const LoggedInUserWeekList = (props: Props) => {
+  const fetchLikes = async () => {
+    return await API.graphql(graphqlOperation(listLikedContents))
+  }
+
+  const { data, status } = useQuery("someKeyName", fetchLikes)
+
+  return (
+    <Weeks>
+      {status === "loading" && <div>Loading data ...</div>}
+
+      {status === "error" && <div>Error fetching bookmarked data</div>}
+
+      {status === "success" &&
+        props.data.map(({ node: week }: { node: ContentfulWeek }) => {
```

Comment:
Prefer destructuring data from props earlier on, and do the combining of the data outside the return function.